utility/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+kt_android_library(
+    name = "utility_lib",
+    custom_package = "org.oppia.util",
+    srcs = glob(["src/main/java/org/oppia/util/**/*.kt"],
+                exclude = ["src/main/java/org/oppia/util/logging/firebase/*.kt"]),
+    resource_files = glob(["src/main/res/**/*.xml"]),
+    manifest = "src/main/AndroidManifest.xml",
+    deps = [
+        ":dagger",
+        "//model:model_lib",
+        "//firebase:crashlytics_lib",
+        "//firebase:crashlytics_deps",
+        artifact("com.google.firebase:firebase-analytics:17.4.4"),
+        artifact("com.google.firebase:firebase-crashlytics:17.1.1"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-core"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-android"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-test"),
+        artifact("com.google.truth:truth"),
+        artifact("androidx.appcompat:appcompat"),
+        artifact("com.github.bumptech.glide:glide"),
+        artifact("com.github.bumptech.glide:compiler"),
+        artifact("com.caverock:androidsvg-aar"),
+        artifact("androidx.lifecycle:lifecycle-livedata-ktx"),
+        artifact("com.crashlytics.sdk.android:crashlytics:2.9.8"),
+        artifact("io.fabric.sdk.android:fabric:1.4.7"),
+        artifact("com.google.gms:google-services:4.3.3"),
+        artifact("com.android.support:support-annotations"),
+        artifact("androidx.test.ext:junit"),
+        artifact("org.mockito:mockito-core:2.19.0"),
+    ],
+    visibility = ["//visibility:public"]
```

Comment:
Don't we need `,` in the end?

Reply:
I just checked and it is convention to include a final comma! I will add that now! Thanks!

Reply:
Thanks, check other places too if you found missing anywhere add there too.

Reply:
Okay!